WebInstall/explainer_cross_domain.md

```diff
@@ -0,0 +1,259 @@
+    * `AbortError`: The installation (prompt) was closed/cancelled.
+
+
+![Promises resolve/reject flow](./installPromises.png) 
+
+
+#### **Signatures of the `install` method (cross-origin)**
+The cross-origin part of the Web Install API consists of the extension to the navigator interface with an `install` method. This method receives:
+* `manifest_id`: decalres the specific application to be installed. If the content to be installed is not content with a manifest, a `install_url` parameter can be specified.
+* optional [parameters](#parameters).
+
+Unless the UA decides to [gate this functionality behind installation](#gating-capability-behind-installation), the behaviour between calling the `install` method on a tab or on an installed application should not differ. The install method can be used in two different ways.
+
+1. `navigator.install(<manifest_id>)`: The method receives a parameter which is a [manifest id](https://w3c.github.io/manifest/#id-member) to a web app to install. This will prompt for installation of the app if the requesting origin has installation permissions (see [security section](#integration-with-the-permissions-api)). This is the most common use of the API.
+
+2. `navigator.install(<manifest_id>, <params>)`: This signature of the method includes the optional parameters. These parameters alter how the app is installed and are defined in an object. More information about the parameters is found in the [Parameters](#parameters) subsection of this specification.
+
+#### **Parameters**
+
+The `navigator.install` call can receive an object with a set of parameters that specify different installation behaviours for the app. It is also a way to future-proof the API if additional data were required with the call.
+* **install_url**: Provides a url to install if the specified `manifest_id` can't be found. The UA can attempt to install this origin instead. 
+* **referral-info**: this parameter takes the form of an object that can have arbitrary information required by the calling installation domain. This information can later be retrieved inside the installed application via the [Acquisition Info API](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/AcquisitionInfo/explainer.md). 
+
+#### **Installing the web app**
+
+To install a same domain web site/app, the process is as follows:
+1. Origin site that triggers the installation must have installation permissions as it tries to install a cross-origin app.
+2. target site/app must comply with *[installability criteria](#installability-criteria), if any*.
+3. If the target content is a web app with a manifest, check if the domain is in the list of [allowed origins](#install-sources-manifest-field) to install said content. If the target content is not a web app, the UA may offer different UX that informs the user that the target might not integrate with the host OS.
```

Comment:
Not completely sure what you're trying to say with 3 with regard to unmanifested apps.  Are you saying that the UA may choose to not install them, or are you saying that the lack of a manifest may mean that certain integrations with the OS may not be possible without a manifest?